modyn/config/examples/modyn_config.yaml

```diff
@@ -45,6 +45,90 @@ storage:
         ignore_last_timestamp: false,
         file_watcher_interval: 5,
         selector_batch_size: 2000000,
+      },
+      {
+        name: "yearbook",
+        description: "Yearbook Dataset from Wild-Time",
+        version: "0.0.1",
+        base_path: "/datasets/yearbook",
+        filesystem_wrapper_type: "LocalFilesystemWrapper",
+        file_wrapper_type: "BinaryFileWrapper",
```

Comment:
Why do we use the binary file wrapper here? That one was built for criteo and I wonder why it maps to image data. Shouldn't we use the single sample file wrapper with regular image functionality?

Reply:
Yearbook images are extremely small (32x32, B/W) so having one image per file is not efficient. Since the images and labels have a fixed size, I used BinaryFIleWrapper.

Reply:
I had tried both versions and BinaryFileWrapper was faster

Reply:
I added a brief comment to the README